### ISSUE TITLE
wallets produced by generate-wallet do not load: jwk wrapping object

### ISSUE THREAD

**Issue by xloem** [ardriveapp/ardrive-cli#248]
it looks like generate-wallet outputs wallets wrapped by `{"jwk":}`. this produces 'undefined' errors when the output wallets are later loaded.

example:
```
$ ardrive generate-wallet -s 'this is an example twelve word seed phrase that you could use' > ardrive_wallet.json
$ ardrive get-address --wallet-file ardrive_wallet.json
TypeError: Cannot read property 'replace' of undefined
```

likely very easy to patch but i thought i'd open an issue because it seems very strange that such basic functionality isn't working. how is everybody generating wallet files?

**Comment by fedellen** [comment]
Thanks for this report @xloem!

I've reproduced this on my end pretty easily. We're tracking this internally as PE-1017. Sorry you ran into this in production

As a temp workaround you could edit your wallet's `.json` structure to just remove that extra `{"jwk":}` and then it seems to work out as a valid wallet.

> how is everybody generating wallet files?

I think most of the team generates them with the ArConnect browser extension or directly with `arweave-js`. Hopefully we can get this fixed and also expand wallet support on the CLI to work without seed-phrases as well.

**Comment by xloem** [comment]
thanks fedellen

**Comment by agsuy** [comment]
Fix merged on 1.9.0
